crates/sui-framework/sources/package.move

```diff
@@ -310,6 +310,16 @@ module sui::package {
         }
     }
 
+    #[test_only]
+    public fun get_upgrade_policy(ticket: &UpgradeTicket): u8 {
+        ticket.policy
+    }
+
+    #[test_only]
+    public fun get_upgrade_digest(ticket: &UpgradeTicket): vector<u8> {
+        ticket.digest
+    }
```

Comment:
note, I actually landed non-test only accessors in `main` (called `ticket_policy` and `ticket_digest` I believe), because of a partner request, so you could now use those -- sorry for the churn there!

Reply:
Awesome! I think those changes haven't made it through yet, but I'll squash and rebase the uber-PR hopefully either tonight or tomorrow morning, and I'll update this once those changes trickle up :)

Reply:
Updated!